R/cast.R

```diff
@@ -133,6 +133,19 @@ vec_cast.default <- function(x, to) {
   stop_incompatible_cast(x, to)
 }
 
+# Cast `x` to `to` but only if they are coercible
+vec_coercible_cast <- function(x, to, ..., x_arg = "", to_arg = "") {
+  if (!missing(...)) {
```

Comment:
I'm curious: is the check here really required?

Reply:
It's not required, that's just a micro-optimisation. I don't recommend doing this except in very low level packages.